setup.py

```diff
@@ -119,6 +119,10 @@
     'databricks-sdk==0.23.0',
 ]
 
+extra_deps['alipan'] = [
+    'AliPCS-Py>=0.8',
```

Comment:
Can you please restrict the version to a major version? For example, `AliPCS-Py>=0.8,<1`?

Reply:
done.